cg/server/ext.py

```diff
@@ -22,6 +24,9 @@
     SampleRunMetricsService,
 )
 from cg.services.sample_service.sample_service import SampleService
+from cg.services.store.rna_dna_collections_service.rna_dna_collections_service import (
```

Comment:
Added this here because that was done for the `sample_service`. But why should the services be imported here? 🤔